package.json

```diff
@@ -62,7 +62,7 @@
     "clean": "node ./scripts/clean",
     "generate-html-files": "node ./scripts/generate-html-files",
     "build": "npm-run-all -s clean webpack generate-html-files",
-    "pretest": "npm run build",
+    "pretest": "npm run lint && npm run build",
```

Comment:
Since `npm-run-all` is already a dependency, what do you think about using `run-s lint build` or more explicit `npm-run-all -s lint build` like in the `build` task.

Reply:
Good suggestion, applied.

There is also another package, which claims to do the same, but faster: https://github.com/coderaiser/redrun#speed-comparison